### PR TITLE
VP-1450 Implement user removing ability

### PR DESCRIPTION
**Pull request by trueboroda**
For confirmation using Bootstrap-Vue ModalPlugin functionality.  this.$bvModal.msgBoxConfirm()

### CODE REVIEW

File: locales/en.default.json [hunk 1]
```diff
@@ -121,6 +121,11 @@
         "delete": "Delete",
         "text-above": "",
         "text-below": ""
+      },
+      "confirm-delete-modal": {
+        "ok": "YES",
+        "cancel": "NO",
+        "message": "Please confirm that you want to delete the user \"{0}\"."
```
Comment (ilyawzrd) [inline locales/en.default.json]: I think we should do something like this for confirmation (at least change phrases)
image

File: client-app/src/pages/account/views/AccountUsers.vue [hunk 2]
```diff
@@ -108,12 +108,24 @@ export default class AccountUsers extends Vue {
...
+      centered: true
+    })
+      .then(value => {
+        if(value) {
+          this.deleteUser(user.id!);
+        }
+      })
+      .catch(err => {
+        // An error occurred
+        console.log(err);
```
Comment (ilyawzrd) [inline client-app/src/pages/account/views/AccountUsers.vue]: do we need to catch errors here? i think we have errorHandler in initialization service

File: locales/en.default.json (hunk 1 above)
Comment (t13ka) [reply locales/en.default.json]: Agree with Ilya

Comment (trueboroda) [reply locales/en.default.json]: Done
https://virtocommerce.atlassian.net/wiki/spaces/ODT/pages/216006668/Manage+Users

File: client-app/src/pages/account/views/AccountUsers.vue (hunk 2 above)
Comment (trueboroda) [reply client-app/src/pages/account/views/AccountUsers.vue]: I agree

**Pull request merged**